src/Symfony/Component/Notifier/Bridge/Esendex/CHANGELOG.md

```diff
@@ -4,11 +4,11 @@ CHANGELOG
 5.3.0
 -----
 
- * The bridge is not marked as `@experimental` anymore
-* [BC BREAK] Changed signature of `EsendexTransport::__construct()` method from:
-  `public function __construct(string $token, string $accountReference, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)`
-  to:
-  `public function __construct(string $email, string $password, string $accountReference, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)`
+ * The bridge is not marked as `@experimental` anymore.
```

Comment:
Instead of going back and forth on this, I've documented the conventions in the docs: https://github.com/symfony/symfony-docs/pull/14830/files

Reply:
Done